src/io/xpi.js

```diff
@@ -39,6 +39,10 @@ export class Xpi extends IOBase {
     if (/\/$/.test(entry.fileName)) {
       return;
     }
+    if (!this.shouldScanFile(entry.fileName)) {
+      log.debug(`not including this for scanning: ${entry.fileName}`);
```

Comment:
This is a serious nitpick but can we remove "this" from the sentence or rephrase it?

```js
`Not included for scanning: ${entry.fileName}`
```

would read easier to me were I a dev debugging stuff.